schema/utils.go

```diff
@@ -35,6 +38,23 @@ func isNumber(n interface{}) (float64, bool) {
 	}
 }
 
+// isTime takes an interface as input, and returns a time.Time if the type is
+// a string and is formatted with RFC3339
+func isTime(t interface{}) (time.Time, bool) {
+	var t2 time.Time
+	var isTime bool
+	if tStr, ok := t.(string); ok {
+		t2, err := time.Parse(time.RFC3339, tStr)
```

Comment:
You should support all the formats defined in the [`time` validator](https://github.com/rs/rest-layer/blob/master/schema/time.go#L38).

The problem here, is that we don't have access to the schema, so we don't know if the field is supposed to be a time, and if user defines custom formats on the time field, these won't be supported here.